frontend/src/utils/api.js

```diff
@@ -1,2 +1,73 @@
 //TODO: Add API calls here!
 require("node-fetch");
+const host = "http://localhost:5000/api";
+
+async function getEndpoint(endPoint, dataKey, additonal_headers = null) {
+  try {
+    let headers = { ...additonal_headers, "Content-Type": "application/json" };
+    let response = await fetch(`${host}/${endPoint}`, {
+      method: "GET",
+      headers
+    });
+    let responseJson = await response.json();
+    return dataKey === "" || responseJson.success === false
+      ? responseJson.result
+      : responseJson.result[dataKey];
+  } catch (error) {
+    console.error(error);
```

Comment:
let's have a more robust error handler

Reply:
Yes, preferably a new function in a new file in `utils` that handles errors.